src/overview/components/Overview.jsx

```diff
@@ -18,7 +19,7 @@ class Overview extends React.Component {
                 ref='inputQuery'
             >
             </input>
-            <ResultList searchResult={this.props.searchResult} />
+            {this.props.waitingForResults==0 ? <ResultList searchResult={this.props.searchResult} />: <LoadingIndicator />  }
```

Comment:
Weird? This should be normal javascript:
`{this.props.waitingForResults ? <LoadingIndicator /> : <Resultlist searchResult={this.props.searchResult} />}`

Are you sure the problem was not something else?